### PR TITLE
fix for #1580

### PR DESCRIPTION
**Pull request by stephenwaite**

### CODE REVIEW

File: interface/patient_file/report/custom_report.php [hunk 1]
```diff
@@ -607,8 +607,8 @@ function postToGet($arin)
                     '/documents/' . $from_pathname . '/' . $from_filename;
                     $to_file = substr($from_file, 0, strrpos($from_file, '.')) . '_converted.jpg';
                 }
-
-                if ($extension != ".pdf") {
+                // adding support for .txt MDM-TXA interface/orders/receive_hl7_results.inc.php
+                if ($extension != (".pdf" || ".txt")) {
                     $image_data = getimagesize($from_file);
```
Comment (bradymiller) [inline interface/patient_file/report/custom_report.php]: Does the getimagesize() function work on a text file?

Comment (bradymiller) [reply interface/patient_file/report/custom_report.php]: lol, I misread the comparison on my review (read as == rather than !=)

**Comment by bradymiller** [comment]
Code looks good (and I am assuming it tests well for you). Will bring it into the codebase and also next 5.0.1 patch. thanks for the fix! -brady

**Comment by stephenwaite** [comment]
cool, thank you Brady

**Pull request merged**